CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Changelog
 
 
-## 0.55.1
+## 0.56.0
```

Comment:
It's a change, we're in version 0, and is not a patch because we're breaking compatibility with possible existing tests/analysis/examples...